packages/amesos2/src/Amesos2_ShyLUBasker_def.hpp

```diff
@@ -88,22 +88,49 @@ ShyLUBasker<Matrix,Vector>::ShyLUBasker(
   typedef Kokkos::OpenMP Exe_Space;
 
   ShyLUbasker = new ::BaskerNS::BaskerTrilinosInterface<local_ordinal_type, slu_type, Exe_Space>();
-  ShyLUbasker->Options.no_pivot      = BASKER_TRUE;
-  ShyLUbasker->Options.symmetric     = BASKER_FALSE;
-  ShyLUbasker->Options.realloc       = BASKER_FALSE;
-  ShyLUbasker->Options.verbose       = BASKER_FALSE;
-  ShyLUbasker->Options.matching      = BASKER_TRUE;
-  ShyLUbasker->Options.matching_type = 0;
-  ShyLUbasker->Options.btf           = BASKER_TRUE;
-  ShyLUbasker->Options.amd_btf       = BASKER_TRUE;
-  ShyLUbasker->Options.amd_dom       = BASKER_TRUE;
+  ShyLUbasker->Options.no_pivot      = BASKER_FALSE;
+  ShyLUbasker->Options.static_delayed_pivot = 0;
+  ShyLUbasker->Options.symmetric      = BASKER_FALSE;
+  ShyLUbasker->Options.realloc        = BASKER_TRUE;
+  ShyLUbasker->Options.verbose        = BASKER_FALSE;
+  ShyLUbasker->Options.prune          = BASKER_TRUE;
+  ShyLUbasker->Options.btf_matching   = 2; // use cardinary matching from Trilinos, globally
+  ShyLUbasker->Options.blk_matching   = 1; // use max-weight matching from Basker on each diagonal block
+  ShyLUbasker->Options.min_block_size = 0; // no merging small blocks
+  ShyLUbasker->Options.amd_dom          = BASKER_TRUE;  // use block-wise AMD
+  ShyLUbasker->Options.use_metis        = BASKER_TRUE;  // use scotch/metis for ND (TODO: should METIS optional?)
```

Comment:
@ndellingwood. You are right when HAVE_SHYLU_NODEBASKER_METIS is false and we run ShuLU-Basker with multiple threads, it will throw an exception with this default. We could set it to false when metis is not enabled, but then it will throw exception if we run with multiple threads and scotch is not enable. I think we prefer METIS.. So maybe, this default is okay?